wgsl/index.bs

```diff
@@ -11277,6 +11287,11 @@ For a given group of invocations:
         in the same [=compute shader stage/workgroup=].
     - For other shader stages, the scope of uniform control flow is all invocations for that
         [=entry point=] in the same [=draw command=].
+    - If the [=language_extension/subgroup_uniformity=] feature is supported,
```

Comment:
It seems the spec should have a normative definition for "uniformity scope", definitions for the cases for workgroup, subgroup, and draw; and then define uniform control flow as being with respect to a given scope.

Also, this section says the definitions are non-normative.  But now they are load-bearing.